src/modules/notes/controllers/createNote.js

```diff
@@ -0,0 +1,11 @@
+import { PROVIDE_VALUES } from '../constants/index.js';
+
+export const createNote = (req, res) => {
+	if (!Object.keys(req.body).length) {
+		return res.status(400).json({
+			success: false,
+			message: PROVIDE_VALUES
+		});
+	}
+	return res.status(201).json(req.body);
```

Comment:
let's try to implement Error Handler to avoid duplication of the `return` statement per controller.
So here will be:
```
if (noParameters) {
  throw new Error(ERROR_CODE, PROVIDE_VALUES)
}
```
And Error Handler will need to take this error and send response to the client